tools/data_integrity/fix_candidate_age.php

```diff
@@ -133,3 +164,22 @@
 if (!$confirm) {
     echo "\n\nRun this tool again with the argument 'confirm' to perform the changes\n\n";
 }
+
+/*
+ * Prints the usage and example help text and stop program
+ */
+function showHelp()
+{
+    echo "*** Fix Candidate Age ***\n\n";
+
+    echo "Usage: php fix_candidate_age.php [confirm | check | help | -h]\n";
+    echo "Example: php fix_candidate_age.php help\n";
+    echo "Example: php fix_candidate_age.php check\n";
+    echo "Example: php fix_candidate_age.php confirm\n\n";
+
+    echo "When the 'check' option is used, the script only detects and reports 
+    miscalculated and NULL ages. 
+    Using the 'confirm' option will apply the necessary corrections to the data.";
```

Comment:
should probably add a `\n` at the end of the help section so that the prompt does not end up right to the last word of the help section.

Reply:
@cmadjar DONE